README.md

```diff
@@ -6,6 +6,7 @@ This monorepo consolidates the following packages:
 
 | Package                              | Version                                                                                                           |
 | :----------------------------------- | :---------------------------------------------------------------------------------------------------------------- |
+| [core-types](packages/core-types/)   | [![version](https://badge.fury.io/js/@nmshd%2fcore-types.svg)](https://www.npmjs.com/package/@nmshd/core-types)   |
```

Comment:
Should we really keep the version? It should be the same for all libraries, so there is no actual information that this column adds.

Reply:
it looks shitty without. so yes.